### ISSUE TITLE
PostgreSql Auto-Complete

### ISSUE THREAD

**Issue by mapes610** [serge-rider/dbeaver#1252]
Hi, 
There is a problem with auto-complete of table names containing upper case letters in the name.

Writing select * from t44b30d98 works well, but
select * from "T44B30D98" cant find anything.

**Comment by serge-rider** [comment]
Probably duplicate of #1232

**Comment by mapes610** [comment]
Hi, 
its not duplicate, I have tried your version 4.0 with repaired #1232 which works just fine, but auto-complete not

**Comment by serge-rider** [comment]
Well, then I just can't reproduce that. Auto-complete works with any case:
![default](https://cloud.githubusercontent.com/assets/6398845/22852899/33ba306a-f058-11e6-9159-17ce1eca83cb.png)


Please provide more details.

**Comment by mapes610** [comment]
Hi, 
I have created table "TESTUSER" which works well. 
If I create table 
create table t9e7d504c as (select * from "TESTUSER")
then autocomplete works just fine:

![dbeaver_2017-02-11_11-01-09](https://cloud.githubusercontent.com/assets/11725433/22853021/7d723e4a-f04a-11e6-8328-66f60e2ff4d9.png)

the same name, but with uppercase
T9E7D504C
![dbeaver_2017-02-11_11-02-12](https://cloud.githubusercontent.com/assets/11725433/22853025/af084418-f04a-11e6-9b26-804df6ab09d7.png)

another example
autocomplete with T03
![dbeaver_2017-02-11_11-16-58](https://cloud.githubusercontent.com/assets/11725433/22853057/c897c984-f04b-11e6-90f1-4e15345b5719.png)


After adding the table I have refreshed list of tables in navigator.
To make sure I have disconnected, but same behavior.
All of these examples tested in Open database object work well, thanks for repair.
